connectivity/tests/world.go

```diff
@@ -12,21 +12,23 @@ import (
 
 // PodToWorld sends multiple HTTP(S) requests to one.one.one.one
 // from random client Pods.
-func PodToWorld() check.Scenario {
-	return &podToWorld{}
+func PodToWorld(domain string) check.Scenario {
```

Comment:
Rather than having to pass the domain to each invocation of `PodToWorld`, let's extract it from `(*check.Test).Context().Params()` in `(*podToWorld).Run` below.

Reply:
It was really good suggestion, I was not happy with supplying domain name this way as well. Changed now.